src/js/components/state/RecipientFYPicker.jsx

```diff
@@ -79,53 +69,25 @@ export default class FYPicker extends React.Component {
         });
     }
 
-    render() {
-        const fy = [];
-        const currentFY = FiscalYearHelper.currentFiscalYear();
-        const earliestFY = FiscalYearHelper.earliestFiscalYear;
-        for (let year = currentFY; year >= earliestFY; year--) {
-            const item = (
-                <li
-                    key={year}
-                    className="fy-picker__list-item">
-                    <button
-                        className="fy-picker__item"
-                        value={year}
-                        onClick={this.clickedYear}>
-                        FY {year}
-                    </button>
-                </li>
-            );
-
-            fy.push(item);
-        }
-
-        const otherFyOptions = fyOptions.map((option) => (
-            <li
-                key={option.name}
-                className="fy-picker__list-item">
-                <button
-                    className="fy-picker__item"
-                    value={option.name}
-                    onClick={this.clickedYear}>
-                    {option.label}
-                </button>
-            </li>
-        ));
+    sortDropdownOptions(key1, key2) {
```

Comment:
When moving forward with a universal picker component we might want to use key values that sort by the defaults sort like all numeric characters or all letters.